packages/store/gas-report.json

```diff
@@ -261,13 +261,13 @@
     "file": "test/Mixed.t.sol",
     "test": "testSetAndGet",
     "name": "set record in Mixed",
-    "gasUsed": 110021
+    "gasUsed": 111113
```

Comment:
very interesting that the gas cost increased for this one (while decreasing for the other tests). @dk1a do you have an intuition around why?

Reply:
pretty sure it's cause `registerSchema` is called in that test (before gas report), which warms the slot, so there's no gas reduction, but there is a call to `getSchema` which is very suboptimal (should be better after https://github.com/latticexyz/mud/pull/1252 and SchemaType->staticLength)